src/controllers/applicationsContentController.js

```diff
@@ -0,0 +1,304 @@
+const cpToken = require(process.cwd() + '/cpToken');
+const constants = require('../utils/constants');
+const path = require('path');
+const fs = require('fs');
+const axios = require('axios');
+const https = require('https');
+
+const filePrefix = config.pathToContent;
+const certPath = path.join(process.cwd(), config.certificate.path);
+const certPassword = config.certificate.password;
+const controlPlaneUrl = config.controlPlaneUrl;
+const token = cpToken.token;
+
+const httpsAgent = new https.Agent({
+    ca: fs.readFileSync(certPath),  
+    rejectUnauthorized: true,
+});
+
+// ***** Load Applications *****
+
+const loadApplications = async (req, res) => {
+    const orgName = req.params.orgName;
+    let html, metaData, templateContent;
+    if (config.mode === constants.DEV_MODE) {
+        metaData = await getMockApplications();
+        templateContent = {
+            applicationsMetadata: metaData,
+            baseUrl: constants.BASE_URL + config.port
+        }
+    }
+    else {
+        console.log('/' + orgName);
+        metaData = await getAPIMApplications();
+        templateContent = {
+            applicationsMetadata: metaData,
+            baseUrl: '/' + orgName
+        }
+    }
+    html = renderTemplate('../pages/applications/page.hbs', filePrefix + 'layout/main.hbs', templateContent, true);
+    res.send(html);
+}
+
+async function getMockApplications() {
+    const mockApplicationsMetaDataPath = path.join(process.cwd(), filePrefix + '../mock/Applications', 'applications.json');
+    const mockApplicationsMetaData = JSON.parse(fs.readFileSync(mockApplicationsMetaDataPath, 'utf-8'));
+    return mockApplicationsMetaData.list;
+}
+
+async function getAPIMApplications() {
+    try {
+        const response = await axios({
+            method: 'GET',
+            url: controlPlaneUrl + '/applications',
+            headers: {
+                Authorization: `Bearer ${token}`,
+            },
+            httpsAgent
+        });
+        return response.data.list;
+    } catch (error) {
+        console.error('Error fetching applications:', error.message);
+    }
+}
+
+// ***** Load Throttling Policies *****
+
+const loadThrottlingPolicies = async (req, res) => {
+    const orgName = req.params.orgName;
+    let html, metaData, templateContent;
+    if (config.mode === constants.DEV_MODE) {
+        metaData = await getMockThrottlingPolicies();
+        templateContent = {
+            throttlingPoliciesMetadata: metaData,
+            baseUrl: constants.BASE_URL + config.port
+        }
+    }
+    else {
+        metaData = await getAPIMThrottlingPolicies();
+        templateContent = {
+            throttlingPoliciesMetadata: metaData,
+            baseUrl: '/' + orgName
+        }
+    }
+    html = renderTemplate('../pages/add-application/page.hbs', filePrefix + 'layout/main.hbs', templateContent, true);
+    res.send(html);
+}
+
+async function getMockThrottlingPolicies() {
```

Comment:
Shall we remove the mocks?

Reply:
If we keep mocks, users will be able to view the pages without doing anything. If we remove them, pages will be empty in dev mode. Isn't it better to have the pages in dev even users can't do any functions on the dev mode.